docs/events/timeline-ui-overview.asciidoc

```diff
@@ -132,36 +120,32 @@ For more information about cases, see <<cases-overview, Cases>>.
 You can view, duplicate, export, delete, and create templates from existing Timelines:
 
 . Go to *Investigate* -> *Timelines*.
-. Click the *All actions* icon in the relevant row, then select the appropriate action:
+. Click the *All actions* icon in the desired row, then select an action:
 
-* *Create template from timeline* (see <<timeline-templates-ui>>)
+* *Create template from timeline* (refer to <<timeline-templates-ui>>)
 * *Duplicate timeline*
-* *Export selected* (see <<import-export-timelines>>)
+* *Export selected* (refer to <<import-export-timelines>>)
 * *Delete selected*
 
-TIP: To perform the same action on multiple Timelines, select the Timelines and
-then the required action from the *Bulk actions* menu.
+TIP: To perform an action on multiple Timelines, first select the Timelines,
+then select an action from the *Bulk actions* menu.
 
 [discrete]
 [[import-export-timelines]]
 == Export and import Timelines
 
-You can import and export Timelines, which enables importing Timelines from one
-{kib} space or instance to another. Exported Timelines are saved in an
-http://ndjson.org[`ndjson`] file.
+You can export and import Timelines, which enables you to share Timelines from one
+{kib} space or instance to another. Exported Timelines are saved as
+http://ndjson.org[`ndjson`] files.
 
-. Go to *Investigate* -> *Timelines*.
-. To export Timelines, do one of the following:
-
-* To export one Timeline, click the *All actions* icon in the relevant row and
-then select *Export selected*.
-* To export multiple Timelines, select all the required Timelines and then click
-*Bulk actions* -> *Export selected*.
+To export timelines:
```

Comment:
Not sure what exactly is happening here, but the bullets and the Note admonition aren't rendering correctly:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/87339667/156470691-ac3046f9-2e05-4861-ae83-9bd7cc809299.png">

Reply:
Adding a new line between "To export Timelines" in line 141 and the first step in the procedure in line 142 should fix the list issue. Same with the second set of procedures for importing Timelines. The note likely also needs a plus sign (`+`) after it to close the note block.